.github/config/en-custom.txt

```diff
@@ -803,3 +803,5 @@ customizable
 implementers
 azureWorkloadIdentity
 untrusted
+SecretConfig
+ConfigMaps
```

Comment:
nit: it's a better idea to surround these with backticks in the main document rather than adding them to spellcheck.

Reply:
SecretConfig is also in a heading so we may need it here as well.I updated it for ConfigMaps